frontend/src/views/CalendarView.vue

```diff
@@ -124,9 +55,8 @@ const getRemoteEvents = async (from, to) => {
 
 /**
  * Retrieve new events if a user navigates to a different month
- * @param dateObj
  */
-const onDateChange = async (dateObj) => {
+const onDateChange = async (dateObj: Time<Dayjs>) => {
```

Comment:
iirc the Dayjs type doesn't properly mixin the timezone property, we might want to use the type I created for dayJsKey in keys.ts.

Reply:
Thank you, I tried to do that, but unfortunately it creates issues with dj() itself, because it only accepts string, number, Date or Dayjs.

![image](https://github.com/user-attachments/assets/e484bd87-c0b8-459a-ab96-274784b22772)

Reply:
Resolving this for now, but I'll keep it in mind if we run into tz issues at this place.